src/app/quiz/components/GetStartedButton/index.tsx

```diff
@@ -0,0 +1,59 @@
+'use client';
+
+import clsx from 'clsx';
+import Image from 'next/image';
+import { HTMLAttributes } from 'react';
+
+import { Text } from '@/common/components';
+import { ICON_URL } from '@/common/constants';
+import { bgColor } from '@/common/foundation';
+
+interface GetStartedButtonProps
+  extends Omit<HTMLAttributes<HTMLButtonElement>, 'className'> {
+  className?: string;
+  isCommentEmpty: boolean;
+}
+
+export function GetStartedButton({
+  className,
+  isCommentEmpty,
+}: GetStartedButtonProps) {
+  return (
+    <button
+      className={clsx(
+        className,
+        bgColor['primaryDefault'],
+        'w-full',
+        'rounded-16px',
+        'px-16px',
+        'py-20px'
+      )}
```

Comment:
```suggestion
    <button
      className={clsx(
        'w-full rounded-16px px-16px py-20px', 
        bgColor["primaryDefault"],
        className
      )}
```

은 어떠신가요? 그리고 className을 어떤걸 받아오나요

Reply:
아 그러네요 하나로 합치겠습니다!
className은 해당 컴포넌트를 사용하는 쪽에서 margin같은 속성을 부여하려고 뚫어놨어요!

Reply:
https://github.com/depromeet/toks-web/pull/304/commits/a61f20255334f7123da74abb23de68b52855ec38
이 부분도 수정했어요!